classes/PodsUI.php

```diff
@@ -3593,10 +3681,36 @@ public function filters_popup() {
 								</label>
 
 								<span class="pods-ui-posts-filter<?php esc_attr_e( ( empty( $start ) && empty( $end ) ) ? ' pods-hidden' : '' ); ?>">
-								<?php echo PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field ); ?>
+								<?php
+								// Prevent p div issues.
+								echo str_replace(
+									array(
+										'<div',
+										'</div>',
+									),
+									array(
+										'<span',
+										'</span>',
+									),
+									PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field )
+								);
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 749f5464b1029fe902dc5ffb6ee7ecbb</sub>